week_1/prob_4.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+
+# Input Arrays with i-th user having ipAddress "ipAddrs[i]", username "userNames[i] and publickey
+publicKey[i]
+
+ipAddrs=()
+userNames=()
+publicKey=()
+
+
+len=${#ipAddrs[@]}
+
+echo "enter remote server's user name"
+read remoteUserNam
+echo "enter remote server's ip address"
+read remoteIpAddr
+
+
+for i in $(seq 0 $((len-1))); 
+do
+
+ipAddr=${ipAddrs[$i]}
+userName=${userNames[$i]}
+publicKey=${publicKeys[$i]}
```

Comment:
from where you are getting this public key? As discussed you should get path of public file and then you have to check it